src/uwtools/drivers/driver.py

```diff
@@ -323,19 +323,21 @@ def _run_via_local_execution(self):
     # Private helper methods
 
     @property
-    def _resources(self) -> dict[str, Any]:
+    def _run_resources(self) -> dict[str, Any]:
         """
         Returns platform configuration data.
         """
         try:
             platform = self._config["platform"]
         except KeyError as e:
             raise UWConfigError("Required 'platform' block missing in config") from e
+        threads = self._driver_config.get("execution", {}).get("threads")
         return {
             "account": platform["account"],
             "rundir": self._rundir,
             "scheduler": platform["scheduler"],
             "stdout": "%s.out" % self._runscript_path.name,  # config may override
+            **({"threads": threads} if threads else {}),
```

Comment:
Optionally returning a `threads` value allows the batch resource request to include e.g. `cpus-per-task` for Slurm, `ompthreads` for PBS, etc.